src/v3/README.md

```diff
@@ -0,0 +1,29 @@
+# User Agent Plugin V3
+
+We want to move user agent parsing out of posthog-js for several reasons
+
+1. it is a significant portion of the bundle size
+2. so, we're having to do code golf which makes it hard to maintain
```

Comment:
What is code golf?

Reply:
https://en.wikipedia.org/wiki/Code_golf
had to google it.